src/Repository/ActiveCampaignChannelRepositoryInterface.php

```diff
@@ -11,6 +11,4 @@
  */
 interface ActiveCampaignChannelRepositoryInterface extends ActiveCampaignResourceRepositoryInterface
 {
-    /** @return ChannelInterface[] */
-    public function findAllEnabledForActiveCampaign(): array;
 }
```

Comment:
This should be removed. SO it should be changes also the ChannelRepository in tests/Application and changed the the installation documentation.

Reply:
Ok, i think i got what you mean. Take a look at the last commit

Reply:
Perfect!